e2e_test/loadScript/fixtures/api.json

```diff
@@ -0,0 +1,624 @@
+{
```

Comment:
That's a huge fixture... Isn't there a risk of frequent changes needed to the file?

Reply:
There is :( Currently I don't have a better idea for testing e2e.